upgrade.js

```diff
@@ -0,0 +1,28 @@
+export const upgradeScripts = [
+	function v2_1(context, props) {
+		const result = {
+			updatedConfig: null,
+			updatedActions: [],
+			updatedFeedbacks: [],
+		}
+		let config = props.config
+		if (config.url === undefined) {
```

Comment:
`props.config` can be undefined depending in what needs upgrading. You should check that before trying to use it

Reply:
Thanks, fixed.